server/model/status_page.js

```diff
@@ -46,7 +47,14 @@ class StatusPage extends BeanModel {
      */
     static async renderHTML(indexHTML, statusPage) {
         const $ = cheerio.load(indexHTML);
-        const description155 = statusPage.description?.substring(0, 155) ?? "";
+
+        let description155 = "";
+        if (statusPage.description) {
+            description155 = marked(statusPage.description)
+                .replace(/(<([^>]+)>)/gi, "")
+                .trim();
+            description155 = description155.substring(0, 155);
+        }
```

Comment:
Continuing https://github.com/louislam/uptime-kuma/pull/4968/files/9860aef1b876b9d9d5fa62d1f602f6c0f37d7821#r1695183786
Lets simply this.
Good point on the optional chaining, overlooked that one.

```suggestion
        const description155 = marked(statusPage.description ?? "")
            .replace(/(<([^>]+)>)/gi, "")
            .trim()
            .substring(0, 155);
```